templates/README.md

```diff
@@ -67,6 +67,12 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 {{ if has (ds "config") "usage" }}
 ## Usage
 
```

Comment:
We should only do this for terraform projects. See example in this file where we check for `main.tf`